_posts/2023-03-15-trace-analytics-jaeger.md

```diff
@@ -0,0 +1,43 @@
+
+## Jaeger
+
+One of the most popular open source solutions for distributed tracing is [Jaeger](https://www.jaegertracing.io/). Jaeger is an open source, end to end solution, hosted by the Cloud Native Computing Foundation (CNCF). Jaeger SDKs are OpenTelemetry (OTEL) based and support multiple open source data stores, like Cassandra and OpenSearch, for storing traces. OpenSearch now provides the option for visualizing traces in the form of the OpenSearch Dashboards Trace analytics solution, which is the native visualization tool included in OpenSearch.
+
+## Trace analytics
+
+OpenSearch provides great support for log analytics and observability use cases. OpenSearch has added support for analyzing distributed tracing data with its Observability feature, starting with version 1.3. Using Observability, you can analyze the crucial Rate, Error, Duration (RED) metrics in their trace data. You can also analyze various components of your system for things like latency and errors, and pinpoint services that need attention.
+
+OpenSearch trace analytics launched with support for OTEL compliant trace data provided by Data Prepper in OpenSearch. To widen the support for more popular trace formats used by developers, OpenSearch recently added support for Jaeger trace data. With this newly added support added in OpenSearch version 2.5, you can now analyze your Jaeger trace data stored in OpenSearch using the Trace analytics feature in OpenSearch Observability.
+
+You can now filter traces and determine exactly which spans are showing errors and narrow issues down to relevant logs quickly. You can also benefit from the same feature rich analysis capabilities around RED metrics, contextual linking of traces, and spans, to their related logs that are available for the Data Prepper trace data.
+
+Keep in mind that OTEL and Jaeger have several differences between their formats, as outlined [here](https://opentelemetry.io/docs/reference/specification/trace/sdk_exporters/jaeger/).
+{: note. }
+
+## Try it out
+
+To try out this new feature, see the following [guide](https://opensearch.org/docs/latest/observing-your-data/trace/trace-analytics-jaeger/), which includes a docker compose file that shows you how to add sample data using the Jaeger hot rod demo, and then visualize it using trace analytics. In order to enable this feature, the `--es.tags-as-fields.all=true` flag needs to be set, as described [here](https://github.com/jaegertracing/jaeger/issues/1299). This is due to a limitation tracked on the following [Github issue](https://github.com/opensearch-project/OpenSearch-Dashboards/issues/657).
```

Comment:
```suggestion
To try out this new feature, see the [Analyzing Jaeger trace data](https://opensearch.org/docs/latest/observing-your-data/trace/trace-analytics-jaeger/) documentation. The documentation includes a docker compose file that shows you how to add sample data using the Jaeger hot rod demo and then visualize it using trace analytics. To enable this feature, you need to set the `--es.tags-as-fields.all` flag to `true`, as described in the related [GitHub issue](https://github.com/jaegertracing/jaeger/issues/1299). This is necessary because of an [OpenSearch Dashboards limitation](https://github.com/opensearch-project/OpenSearch-Dashboards/issues/657).
```